Source/FullScreenPokemon.ts

```diff
@@ -3832,23 +3832,44 @@ module FullScreenPokemon {
 
         /**
          * Opens the context menu within the Pokedex menu for the selected Pokemon.
+         *
+         * @param settings   Contains the selected Pokemon.
          */
         openPokemonMenuContext(settings: any): void {
+            var moves: BattleMovr.IMove[] = settings.pokemon.moves,
+                options: any[] = [],
+                move: IMoveSchema,
+                i: number;
+
+            for (i = 0; i < moves.length; i += 1) {
+                move = this.MathDecider.getConstant("moves")[moves[i].title];
+                if (move.partyActivate) {
+                    options.push({
+                        "text": moves[i].title.toUpperCase(),
+                        "callback": move.partyActivate(this.player, settings.pokemon)
```

Comment:
This is calling `move.partyActivate(...)`, getting the result, and putting it in the object under `"callback"`.

You want to make a new Function that, when called, calls `move.partyActivate(...)`.